test/Swan.Test/JsonFormatterDynamicTest.cs

```diff
@@ -0,0 +1,52 @@
+﻿namespace Swan.Test;
+
+using NUnit.Framework;
+using Swan.Formatters;
+using Swan.Test.Mocks;
+using System.Text.Json;
+
+[TestFixture]
+public class JsonFormatterDynamicTest
+{
+    [Test]
+    public void WithJsonTextNullOrWitheSpace_ReturnsNull()
+    {
+        var jsonText = string.Empty;
+        var jsonObject = jsonText.JsonDeserialize();
+
+        Assert.IsNull(jsonObject);
+    }
+
+    [Test]
+    public void WithJsonText_ReturnsObject()
+    {
+        var jsonText = "{\r\n    \"Name\": \"Merlina\"\r\n}";
+        var jsonObject = jsonText.JsonDeserialize();
+
+        Assert.AreEqual("Merlina", jsonObject?.Name);
+    }
+
+    [Test]
+    public async Task WithNullStream_ReturnsObject()
```

Comment:
Can you change the name of the test pls, it says that you are testing that it returns an object but instead you are testing that it returns null

Reply:
Also you are validating a string with `Name` prop, if you want to check an object, you should use `typeof(jsonObject)`, maybe it is correct as it is, just rename the function, is not doing what you are saying, keep in mind the `Principle of Least Surprise` 😁